packages/core/src/internal/base/css-gap.base.spec.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { LitElement, html } from 'lit-element';
+import { createTestElement, removeTestElement, componentIsStable } from '@cds/core/test/utils';
+import { applyCSSGapShim } from './css-gap.base.js';
+
+class DemoComponent extends LitElement {}
+
+// register the element just for testing purposes
+window.customElements.define('cds-css-gap-shim', applyCSSGapShim(DemoComponent));
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'cds-css-gap-shim': DemoComponent;
+  }
+}
+
+describe('CSS Flex Gap Shim', () => {
+  let testElement: HTMLElement;
+  let component: DemoComponent;
+
+  beforeEach(async () => {
+    testElement = await createTestElement(html`<cds-css-gap-shim></cds-css-gap-shim>`);
+    component = testElement.querySelector<DemoComponent>('cds-css-gap-shim');
+  });
+
+  afterEach(() => {
+    removeTestElement(testElement);
+  });
+
+  it('should only apply css gap shim attr if not supported', async () => {
+    await componentIsStable(component);
+    expect(component.getAttribute('_nfg')).toBe(null);
```

Comment:
I have a test to ensure the shim is not applied to our default test env with chrome. I couldn't think of an easy way to test for when it should be applied since the check is specific to the browser.

Reply:
Could you mock `browserFeatures.supports.flexGap` in a test?

Reply:
Was able to cast/override the support value to check the attr being set